packages/medusa/src/services/swap.ts

```diff
@@ -394,8 +428,14 @@ class SwapService extends BaseService {
     })
   }
 
-  async processDifference(swapId) {
-    return this.atomicPhase_(async (manager) => {
+  /**
```

Comment:
thought(non-blocking): I think that this method could be splitted in multiple sub method for the different cases. But not here. wdyt?